chart/skywalking/values.yaml

```diff
@@ -37,6 +37,7 @@ oap:
     tag: null  # Must be set explicitly
     pullPolicy: IfNotPresent
   storageType: null
+  initEs: true # Whether need to initial ES
```

Comment:
Please add this to doc?

Reply:
I added the description about this argument in two md files. Please review.